newdemojail.yaml

```diff
@@ -0,0 +1,772 @@
+
+  paroleofficer:
+    fqn: "general.person"
+    entitySetName: "DemoParoleOfficers"
+    propertyDefinitions:
+      nc.SubjectIdentification:
+        type: "nc.SubjectIdentification"
+        column: "OfficerPhoneNo"
+      nc.PersonGivenName:
+        type: "nc.PersonGivenName"
+        column: "OfficerPhoneNo"
+      nc.PersonMiddleName:
+        type: "nc.PersonMiddleName"
+        column: "OfficerPhoneNo"
+      nc.PersonSurName:
+        type: "nc.PersonSurName"
+        column: "OfficerPhoneNo"
+    conditions:
+    - !<conditions.BooleanRegexCondition>
+      column: "HearingType"
+      pattern: "Parole"
+      reverse: false
+    name: "paroleofficer"
+
+  probationofficer:
+    fqn: "general.person"
+    entitySetName: "DemoProbationOfficers"
+    propertyDefinitions:
+      nc.SubjectIdentification:
+        type: "nc.SubjectIdentification"
+        column: "OfficerPhoneNo"
+      nc.PersonGivenName:
+        type: "nc.PersonGivenName"
+        column: "OfficerPhoneNo"
+      nc.PersonMiddleName:
+        type: "nc.PersonMiddleName"
+        column: "OfficerPhoneNo"
+      nc.PersonSurName:
+        type: "nc.PersonSurName"
+        column: "OfficerPhoneNo"
+    conditions:
+    - !<conditions.BooleanRegexCondition>
+      column: "HearingType"
+      pattern: "Probation"
+      reverse: false
+    name: "probationofficer"
+
+  paroleattorney:
+    fqn: "general.person"
+    entitySetName: "DemoParoleAttorneys"
+    propertyDefinitions:
+      nc.SubjectIdentification:
+        type: "nc.SubjectIdentification"
+        column: "AttorneyPhoneNo"
+      nc.PersonGivenName:
+        type: "nc.PersonGivenName"
+        column: "AttorneyFirstName"
+      nc.PersonMiddleName:
+        type: "nc.PersonMiddleName"
+        column: "AttorneyMiddleName"
+      nc.PersonSurName:
+        type: "nc.PersonSurName"
+        column: "AttorneyLastName"
+    conditions:
+    - !<conditions.BooleanRegexCondition>
+      column: "HearingType"
+      pattern: "Parole"
+      reverse: false
+    name: "paroleattorney"
+
+  probationattorney:
+    fqn: "general.person"
+    entitySetName: "DemoProbationAttorneys"
+    propertyDefinitions:
+      nc.SubjectIdentification:
+        type: "nc.SubjectIdentification"
+        column: "AttorneyPhoneNo"
+      nc.PersonGivenName:
+        type: "nc.PersonGivenName"
+        column: "AttorneyFirstName"
+      nc.PersonMiddleName:
+        type: "nc.PersonMiddleName"
+        column: "AttorneyMiddleName"
+      nc.PersonSurName:
+        type: "nc.PersonSurName"
+        column: "AttorneyLastName"
+    conditions:
+    - !<conditions.BooleanRegexCondition>
+      column: "HearingType"
+      pattern: "Probation"
+      reverse: false
+    name: "probationattorney"
+
+  attorney:
+    fqn: "ol.occupation"
+    entitySetName: "DemoAttorneys"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "AttorneyPhoneNo"
+      ol.type:
+        type: "ol.type"
+        column: "HearingType"
```

Comment:
Hearing Type? I think this should just write in, "Attorney".

Reply:
This is because i was integrating Probation/ Parole attorneys conditional on the type of hearing - I have since changed this since we are trying to de-transform, and de-condition, but it remains that this value makes sense to write here. If there is an attorney and the hearing is Probation/Parole this will write that the attorney type is Probation/Parole